src/index.js

```diff
@@ -6,29 +6,29 @@ const warn = message => {
   if (console && console.warn) console.warn('Simple Analytics: ' + message || 'Something goes wrong.');
 }
 
-const injectScript = () => {
+const injectScript = (domain) => {
   if (!document) return warn('No document defined.');
   const el = document.createElement('script');
   el.type = 'text/javascript';
   el.async = true;
-  el.src = 'https://cdn.simpleanalytics.io/hello.js';
+  el.src = 'https://' + domain + '/hello.js';
```

Comment:
We are only using `/app.js` for custom domains in the future. Could you update this? So `/hello.js` for `cdn.simpleanalytics.io` and `/app.js` for any other domain.

Reply:
Sure. I will have this done before I go to work.